ecommerce-product-page-main/styled.css

```diff
@@ -367,45 +333,29 @@ ul.list li a:hover{
 
 
 
-.button1{
-
+.discount{
     font-family: 'Kumbh Sans', sans-serif;
     font-weight: 700;
     font-size: 12px;
     color: hsl(26,100%,55%);
     background-color: hsl(26, 100%, 94%);
-    max-width: 45px;
-    max-height: 30%;
+    flex-shrink: 1;

```

Comment:
this flex shrink is to make the "50%" appear smaller to the one its next to yes?

Reply:
no. 

https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-flex-shrink